Description: fix ftbs multiple def of invalid_dircache_entries
Author: Andrew Bauer <zonexpertconsulting@outlook.com>
Bug: https://github.com/Netatalk/Netatalk/issues/124
Bug-Debian: https://bugs.debian.org/957590
Last-Update: 2020-09-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/etc/afpd/directory.h
+++ b/etc/afpd/directory.h
@@ -91,7 +91,7 @@
 #define	AR_UWRITE	(1<<2)
 #define	AR_UOWN		(1<<7)
 
-q_t *invalid_dircache_entries;
+extern q_t *invalid_dircache_entries;
 
 typedef int (*dir_loop)(struct dirent *, char *, void *);
 
